ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportsPage.tsx

```diff
@@ -28,13 +33,14 @@ import { vulnerabilityReportsPath } from 'routePaths';
 import PageTitle from 'Components/PageTitle';
 import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate/EmptyStateTemplate';
 
+import useRunReport from 'Containers/Vulnerabilities/VulnerablityReporting/api/useRunReport';
```

Comment:
I don't particularly appreciate how the relative paths always need to be at the bottom. I generally like splitting up my imports by libraries, then utils, helpers, hooks, modules, and components. Seems like I need to use absolute paths to make it work. I might have rushed this PR and neglected to fix the structure. Let me fix it

Reply:
Oh, I guess I didn't answer your question. I put the `useDeleteModal` in the `hooks` directory because technically it's API-related but also has modal logic. I guess I'm not sure if that constitutes putting it in the `api` directory or not 🤔